package.json

```diff
@@ -5,9 +5,10 @@
   "author": "brandon@formless.xyz",
   "license": "UNLICENSED",
   "dependencies": {
-    "@openzeppelin/contracts": "^4.4.2",
+    "@openzeppelin/contracts": "^4.9.6",
```

Comment:
nit: Let's leave this at the prior version and pin. The reason is these impact the contract build sizes when we init the submodule in our internal BE, and will cause test failures until we synchronize the new dependencies.